accounts/views.py

```diff
@@ -12,6 +12,7 @@
 from django.utils.html import strip_tags
 from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
 
+
```

Comment:
Remove this extra line. There should be a single newline only between the two sections of import. Also, this `accounts/views.py` file shouldn't be a part of this PR as no changes are being done in this file.